plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildInputs/InheritanceTree.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System;
+    using System.Collections.Generic;
+
+    [Serializable]
+    public class InheritanceTree : List<InheritanceTree>
+    {
```

Comment:
oh... should add it

---
In reply to: [112601655](https://github.com/dotnet/docfx/pull/1574#discussion_r112601655) [](ancestors = 112601655)